package.json

```diff
@@ -11,7 +11,7 @@
   },
   "dependencies": {
     "@emotion/react": "^11.11.4",
-    "@emotion/styled": "^11.11.0",
```

Comment:
I don't know why or how but something in your code breaks the home page visually (and doesn't allow you to easily access the drop down).  I assume it's something either to do with the "@emotion/styled" going from version 11.11.0 to 11.11.5 or the CSS file but neither seems to directly change the page so I'm not sure what going on.

Reply:
This was some CSS styling I was trying for the isAvailable toggle, but ended up deleting it. Not sure why this is breaking something however as it's a dependency that isn't being used. 

I went in and deleted it.